Src/Data/InterfaceTheme.lua

```diff
@@ -0,0 +1,20 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+return {
+	Primary = {
+	
+	},
+	Secondary = {
+		Black = Color3.fromRGB(0, 0, 0),
+		LightBlack = Color3.fromRGB(20, 20, 20),
+		White = Color3.fromRGB(255, 255, 255),
+		Pastel = Color3.fromRGB(236, 232, 216),
+	},
+
+	HeaderFont = Font.new("rbxassetid://12187375716", Enum.FontWeight.ExtraBold, Enum.FontStyle.Normal),
+
+	TextFont = Font.fromName(Enum.Font.SourceSans.Name, Enum.FontWeight.Medium, Enum.FontStyle.Normal),
+	TextFontBold = Font.fromName(Enum.Font.SourceSans.Name, Enum.FontWeight.Bold, Enum.FontStyle.Normal),
+	TextFontExtraBold = Font.fromName(Enum.Font.SourceSans.Name, Enum.FontWeight.ExtraBold, Enum.FontStyle.Normal),
+	TextFontItalic = Font.fromName(Enum.Font.SourceSans.Name, Enum.FontWeight.Medium, Enum.FontStyle.Italic),
```

Comment:
Can we use either Inter or Builder Sans for everything?

Reply:
Sur, will thow this on the linear ticket